libvast/src/system/importer.cpp

```diff
@@ -38,21 +38,36 @@ importer_state::importer_state(caf::event_based_actor* self_ptr)
 }
 
 importer_state::~importer_state() {
-  write_state();
+}
+
+caf::error importer_state::bump_boundary() {
+  while (next_id_ >= id_boundary)
+    id_boundary += 10'000'000;
```

Comment:
Please use a power of 2 for this:
```suggestion
    id_boundary += 10Mi;
```